.cargo/config.toml

```diff
@@ -1,2 +1,3 @@
 [build]
-target = "i686-pc-windows-msvc"
```

Comment:
whoops, didn't mean to commit the config file...
i disabled the target because i thought i need to, to be able to target aarch64 (i was wrong btw) and the target-dir is because i cloned the project into a dir that has a shared target dir and i quickly wanted to run the demo nsi script.